demos/text_detection_demo/cpp/README.md

```diff
@@ -12,6 +12,10 @@ The demo shows an example of using neural networks to detect and recognize print
 * `text-recognition-0015`, which is a recognition network for recognizing text. You should add options `-tr_pt_first`, `-m_tr_ss "?0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"` (supported symbols set), `-tr_o_blb_nm "logits"` (to specify output name) and `-dt simple` (to specify decoder type). You can also specify `-lower` option to convert predicted text to lower-case. See model [description](../../../models/intel/text-recognition-0015/README.md) for details.
 * `text-recognition-resnet-fc`, which is a recognition network for recognizing text. You should add option `-tr_pt_first`.
 * `handwritten-score-recognition-0001`, which is a recognition network for recognizing handwritten score marks like `<digit>` or `<digit>.<digit>`.
+* `vitstr-small-patch16-224`, which is a recognition network for recognizing text. You should add options:
+    ```
+    -tr_pt_first, -m_tr_ss 0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ!"#$%&'()*+,-./:;<=>?@[\]^_`{|}~ (supported symbols set), -dt simple (to specify decoder type), -start_index 1 (to process output from provided index) and -pad " " (to use specific pad symbol)
```

Comment:
It seems, that this line contains both CL arguments and comments for them. In that case it uncopypastable.